Components/Order/OrderUpdater.php

```diff
@@ -53,6 +52,11 @@ class OrderUpdater
      */
     private $repoHistory;
 
+    /**
+     * @var PaymentStatusMailValidator
+     */
+    private $validatorPaymentMail;
```

Comment:
just a naming, but isn't it better to name variable like class. The variable name sounds strange